config.toml

```diff
@@ -46,6 +53,14 @@ unsafe = true
 #  src = "img/hexagon.jpg"
 #  desc = "Hexagon"
 
+[taxonomies]
+  # keep default taxonomies
+  category = 'categories'
+  tag = 'tags'
+  # new taxonomies for backblasts
+  ao = 'AOs'
```

Comment:
If I lowercase this here, then when it renders on the page, it will be either `Aos` or `aos` which I think looks weird, like here. But can change to lowercase if you don't think that's a big deal!

<img width="625" alt="image" src="https://user-images.githubusercontent.com/20074219/204166958-2f0f0797-ef61-4544-bd57-4d29f6400d3b.png">

Reply:
Yep makes sense. Thanks!